x/mongo/driver/errors.go

```diff
@@ -116,11 +116,12 @@ func (wce WriteCommandError) Retryable(wireVersion *description.VersionRange) bo
 // WriteConcernError is a write concern failure that occurred as a result of a
 // write operation.
 type WriteConcernError struct {
-	Name    string
-	Code    int64
-	Message string
-	Details bsoncore.Document
-	Labels  []string
+	Name            string
+	Code            int64
+	Message         string
+	Details         bsoncore.Document
+	Labels          []string
+	TopologyVersion *description.TopologyVersion
```

Comment:
Can you send a link to the part of the spec that says that `TopologyVersion` can be included on command errors and write concern errors? I didn't see it in https://github.com/mongodb/specifications/commit/7dd9c008be6edaf65368787d5e91c2ac3aeb95b8.